docs/docs/en/changelog.md

```diff
@@ -0,0 +1,66 @@
+# Change Log
```

Comment:
so the change log only show diff between two version, not including the history?